etc/kayobe/hooks/overcloud-service-deploy/post.d/10-rsyslog.yml

```diff
@@ -0,0 +1 @@
+../../../ansible/rsyslog.yml
```

Comment:
With other playbooks like this I've not added the hook symlink, thinking that not all deployments may want this, and they can opt in easily by adding the symlink.